From def99e39e5f759343634b0f0047343744a0e0c36 Mon Sep 17 00:00:00 2001
From: Xiao Wang <jasowang@redhat.com>
Date: Mon, 27 Nov 2017 07:07:54 +0100
Subject: [PATCH 3/9] slirp: cleanup leftovers from misc.h

RH-Author: Xiao Wang <jasowang@redhat.com>
Message-id: <1511766477-29559-2-git-send-email-jasowang@redhat.com>
Patchwork-id: 77897
O-Subject: [RHEL7.5 qemu-kvm PATCH 1/4] slirp: cleanup leftovers from misc.h
Bugzilla: 1508745
RH-Acked-by: Stefan Hajnoczi <stefanha@redhat.com>
RH-Acked-by: wexu@redhat.com
RH-Acked-by: Miroslav Rezanina <mrezanin@redhat.com>

From: Michael Tokarev <mjt@tls.msk.ru>

There are quite a few leftover declarations in slirp/misc.h.
Remove them.

Signed-off-by: Michael Tokarev <mjt@tls.msk.ru>
(cherry picked from commit a9c34e4485ef0a8a61e88f1c9f9bb87f00ba2c72)
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 slirp/misc.h | 14 --------------
 1 file changed, 14 deletions(-)

diff --git a/slirp/misc.h b/slirp/misc.h
index cc36aeb..ba8beb1 100644
--- a/slirp/misc.h
+++ b/slirp/misc.h
@@ -20,8 +20,6 @@ struct ex_list {
 char *strdup(const char *);
 #endif
 
-void do_wait(int);
-
 #define EMU_NONE 0x0
 
 /* TCP emulations */
@@ -51,21 +49,9 @@ struct emu_t {
     struct emu_t *next;
 };
 
-extern int x_port, x_server, x_display;
-
-int show_x(char *, struct socket *);
-void redir_x(uint32_t, int, int, int);
 void slirp_insque(void *, void *);
 void slirp_remque(void *);
 int add_exec(struct ex_list **, int, char *, struct in_addr, int);
-int slirp_openpty(int *, int *);
 int fork_exec(struct socket *so, const char *ex, int do_pty);
-void snooze_hup(int);
-void snooze(void);
-void relay(int);
-void add_emu(char *);
-void fd_nonblock(int);
-void fd_block(int);
-int rsh_exec(struct socket *, struct socket *, char *, char *, char *);
 
 #endif
-- 
1.8.3.1

